macros/utils/table_operations/insert_rows.sql

```diff
@@ -137,10 +139,16 @@
     {{- return(string_value | replace("'", "''")) -}}
 {%- endmacro -%}
 
-{%- macro render_value(value) -%}
+{%- macro athena__escape_special_chars(string_value) -%}
+    {{- return(string_value | replace("'", "''")) -}}
```

Comment:
@artem-garmash I'm wondering why we need that exactly.

Reply:
for escaping single quote:
```
select 'select ''str_val'' as val' as val
```

Reply:
clear, thanks, I was not really able to seen it.